INSTALL.md

```diff
@@ -0,0 +1,25 @@
+
+## Setup CactusBot:
+```
+git clone https://github.com/cactusdev/cactusbot
+cd cactusbot
+pip3 install -r requirements.txt
+cp config.template.py config.py
+```
+
+Next, open `config.py` with your favorite text editor, and set 
+`USERNAME` and `PASSWORD` to the bots login info, and set `CHANNEL` to your channel's name.
+
+# Usage
+
+Start Sepal:
+
+`npm start`
```

Comment:
This is assuming a 100% fully hosted. We can mention that some where else, but not here.